frontend/pages/doctor/patient-list-overview.tsx

```diff
@@ -0,0 +1,73 @@
+import { Box, Heading, SimpleGrid, useDisclosure } from "@chakra-ui/react";
+import PatientInfoCard from "@frontend/components/doctor/patient-info-card";
+import { serverURL } from "@frontend/config";
+import { Patient } from "@frontend/models/patient";
+import PatientInfoModal from "components/doctor/patient-info-modal";
+import { useState } from "react";
+
+interface AppProps {
+    patientList: Patient[];
+}
+
+export async function getServerSideProps() {
+    let patientList: any = [];
+    try {
+        // <TODO> Get current doctor ID
+        const response: any = await fetch(serverURL + "/patients/getPatientsInfo/3");
+        patientList = await response.json();
+    } catch {}
+    return {
+        props: {
+            patientList,
+        },
+    };
+}
+
+export default function DoctorDashboard({ patientList }: AppProps) {
+    const [selectedPatient, setSelectedPatient] = useState<Patient>({
```

Comment:
i think this basic/default patient can be moved to a utils/constants/models file